src/client/java/dev/creesch/model/WebsocketMessageBuilder.java

```diff
@@ -50,14 +43,31 @@ public static WebsocketJsonMessage createLiveChatMessage(
             );
         }
 
-        Map<String, String> translations =
-            ClientTranslationUtils.extractTranslations(message);
-
         // Can't use GSON for Text serialization easily, using Minecraft's own serializer.
-        String minecraftChatJson = Text.Serialization.toJsonString(
-            message,
-            client.world.getRegistryManager()
-        );
+        // The try block is used as there are instances of `Text` that can't be serialized to JSON
+        Map<String, String> translations;
+        String minecraftChatJson;
+
+        try {
+            translations = ClientTranslationUtils.extractTranslations(message);
```

Comment:
Could move this out of the `try .. catch`

Reply:
completely forgot about this PR. I could, but it makes sense to leave it within the try, considering that in the catch we set it to an empty map. That, to me anyway, makes sense as far as having code where it is used/set.